src/VisualStudio/Core/Def/Diagnostics/VisualStudioDiagnosticAnalyzerProvider.cs

```diff
@@ -23,7 +23,7 @@ internal sealed partial class VisualStudioDiagnosticAnalyzerProvider
 {
     private const string AnalyzerContentTypeName = "Microsoft.VisualStudio.Analyzer";
 
-    internal const string RazorContentTypeName = "Microsoft.VisualStudio.RazorAssembly";
+    internal const string RoslynOopMefContentName = "Microsoft.VisualStudio.RoslynOopMefComponent";
```

Comment:
```suggestion
    internal const string RoslynOopMefContentName = "Microsoft.VisualStudio.RoslynOopMefComponent";
```

Nit: maybe call it RoslynServiceHubMefComponent. Only because I agree that you correctly applied .NET casing rules to the "OOP" but anybody seeing it might be confused.